modules/swagger-parser-v3/src/main/java/io/swagger/v3/parser/ResolverCache.java

```diff
@@ -68,11 +68,10 @@ public class ResolverCache {
     private Map<String, String> externalFileCache = new HashMap<>();
     private List<String> referencedModelKeys = new ArrayList<>();
     private Set<String> resolveValidationMessages;
-
     private final ParseOptions parseOptions;
 
 
-    /*
+	/*
```

Comment:
Would need to use spaces instead of tabs and correct indentation

Reply:
Done!